app/views/records/edit_fields/_rights.html.erb

```diff
@@ -1,4 +1,6 @@
+<% opts = RightsService.select_options %>
+<% opts.unshift ['', ''] %>
```

Comment:
Can you pass `include_blank: true` to `#input` instead of manipulating the collection?